contrib/azure-setup.sh

```diff
@@ -158,7 +169,7 @@ createApplication() {
 
 createServicePrincipal() {
     echo "==> Creating service principal"
-    # Azure CLI 0.10.2 introduced a breaking change, where appId must be supplied with the -a switch
+    # az CLI 0.10.2 introduced a breaking change, where appId must be supplied with the -a switch
```

Comment:
Comment does not match the code (XPlat CLI -> Azure CLI 2.0).

Reply:
Changed to appropriate CLI naming.